assets/js/modules/tagmanager/components/common/AMPContainerSelect.test.js

```diff
@@ -70,6 +72,8 @@ describe( 'AMPContainerSelect', () => {
 		registry.dispatch( STORE_NAME ).setAccountID( accountID );
 		registry.dispatch( STORE_NAME ).receiveGetAccounts( [ account ] );
 		registry.dispatch( STORE_NAME ).receiveGetContainers( containers, { accountID } );
+		registry.dispatch( STORE_NAME ).finishResolution( 'getAccounts', [] );
+		registry.dispatch( STORE_NAME ).finishResolution( 'getContainers', [ accountID ] );
```

Comment:
See above (same applies to the rest of this file).

Reply:
Updated.